runtime/reference/cli/all_commands.md

```diff
@@ -20,27 +36,20 @@ below for more information on each subcommand.
   reports
 - [deno doc](/runtime/reference/cli/documentation_generator/) - generate
   documentation for a module
-- [deno eval](/runtime/reference/cli/eval/) - evaluate provided script
 - [deno fmt](/runtime/reference/cli/formatter/) - format your code
 - [deno info](/runtime/reference/cli/dependency_inspector/) - inspect an ES
   module and all of its dependencies
-- [deno init](/runtime/reference/cli/init/) - create a new project
-- [deno install](/runtime/reference/cli/install/) - install a dependency or a
-  script
+- [deno init](/runtime/reference/cli/init/) - create a new project script
```

Comment:
I don't know the last thing is better than the first

Reply:
not sure what this is in reference to?
Moving the unstable flags to the bottom of the list?

Reply:
"create a new project" vs "create a new project script"

Reply:
oh the script shouldnt be there, it was orphaned from another line, should be fixed now